dts/bindings/i2c/i2c.yaml

```diff
@@ -19,11 +19,9 @@ properties:
     "#address-cells":
       type: int
       category: required
-      description: should be 1.
```

Comment:
Was it intended? Everywhere else we remove `type: int` line.

Reply:
Nope, thanks for catching it.

Reply:
Fixed.